src/app/organization/manage-funds/manage-funds.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="tab-container mat-typography">
+  <h1>Manage Funds</h1>
+  <div>
+    <form #formRef="ngForm" fxLayout="row" fxLayoutAlign="start baseline" fxLayoutGap="20px">
+      <mat-form-field fxFlex="calc(90%-20px)">
+        <textarea matInput placeholder="New Fund"></textarea>
+      </mat-form-field>
+      <button mat-raised-button fxFlex color="primary">
+        <fa-icon icon="plus"></fa-icon>
+      </button>
+    </form>
+  </div>
+  <mat-grid-list cols="2" rowHeight="50px">
+    <mat-grid-tile *ngFor="let manageFundData of manageFundsData; let i=index;">
+      <input disabled="" placeholder={{manageFundData.name}} type="text"/>	
+      <div fxLayout="row" fxLayoutAlign="flex-start">
+        <button mat-button color="primary">
+          <fa-icon icon="edit"></fa-icon>
+        </button>
+      </div>
+    </mat-grid-tile>
+  </mat-grid-list>
+</div>
```

Comment:
Ensure new line at end of each file
```suggestion
</div>

```